test/extended/node_tuning/node_tuning_utils.go

```diff
@@ -0,0 +1,238 @@
+package node_tuning
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
```

Comment:
This is not very clean in my view.  The functions in this file shouldn't need ghnko/gomega at all.  All errors sound be reported (sent back) to the calling code and then (within your g.It() sections call o.Expect(err)... 

I'd propose to keep things clean and remove all o.Expect(err) code from this file.  That way, you'll have more choice what to do with the errors in the future.

Reply:
Remove all o.Expect(err) code  from util file, thanks!